web_advanced_search_wildcard/static/src/js/search.js

```diff
@@ -0,0 +1,12 @@
+odoo.define('web_advanced_search_wildcard', function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var search_filters = require('web.search_filters');
+
+    var _lt = core._lt;
+
+   search_filters.ExtendedSearchProposition.Char.prototype.operators.push(
```

Comment:
This indentation is inconsistent also (sorry to bother you again with this, but you should have fixed it the 1st time 😋)

Reply:
No problem! Fixed now.